src/Symfony/Bundle/FrameworkBundle/Command/CachePoolGetSetCommand.php

```diff
@@ -0,0 +1,107 @@
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Get and/or set cache pool key value.
+ *
+ * @author Antoine Makdessi <amakdessi@me.com>
+ */
+final class CachePoolGetSetCommand extends Command
+{
+    protected static $defaultName = 'cache:pool:get-set';
+
+    private $pools;
+
+    public function __construct(array $pools)
+    {
+        parent::__construct();
+
+        $this->pools = $pools;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Allow to get and/or set key value from cache pools.')
+            ->addArgument('pool', InputArgument::REQUIRED, 'The cache pool from which to get/set a key')
+            ->addArgument('key', InputArgument::REQUIRED, 'The cache key to get/set')
+            ->addArgument('value', InputArgument::OPTIONAL, 'The value to set in the cache pool key')
+            ->addOption('save', null, InputOption::VALUE_OPTIONAL, 'Save the value inside the key of the pool', false)
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> get a value from a given cache pool key.
+
+    %command.full_name% <pool> <key>
+
+The <info>%command.name%</info> set a value on a key on a given cache pool.
+
+    %command.full_name% <pool> <key> <value> --save
+EOF
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $pool = $input->getArgument('pool');
+        if (!\array_key_exists($pool, $this->pools)) {
+            $io->error('Error loading the cache pool.');
+
+            return Command::FAILURE;
+        }
+
+        $cache = $this->pools[$pool];
+
+        $key = $input->getArgument('key');
+
+        if (false !== $input->getOption('save')) {
+            $value = $input->getArgument('value');
+
+            $cacheItem = $cache->getItem($key);
+            $cacheItem->set($value);
+            $isSaved = $cache->save($cacheItem);
+
+            if (!$isSaved) {
+                $io->error('Error saving the value.');
+
+                return Command::FAILURE;
+            }
+
+            $io->success(sprintf('Cache item value "%s" was successfully saved in "%s" "%s".', $value, $pool, $key));
+        }
+
+        if ($cache->hasItem($key)) {
+            $value = $cache->getItem($key)->get();
+
+            $io->success(sprintf('Cache item "%s" "%s" value is "%s".', $pool, $key, $value));
```

Comment:
The value could be anything. You could use `\Symfony\Component\Console\Helper\Dumper` to render it (which uses the VarDumper component if available)

Reply:
i'll try this
but did you meant `CliDumper`?